packages/dappmanager/src/calls/httpsPortal.ts

```diff
@@ -71,6 +71,15 @@ export async function httpsPortalExposableServicesGet(): Promise<
   });
 }
 
+/**
+ * HTTPs Portal: set env value LOCAL_PROXYING
```

Comment:
Explain in more detail what's going on, and why this option exists. Link to the PR in the https repo that adds this